libvast/src/format/zeek.cpp

```diff
@@ -155,13 +155,17 @@ Stream& operator<<(Stream& out, const time_factory& t) {
 }
 
 void print_header(const type& t, std::ostream& out) {
-  VAST_ASSERT(detail::starts_with(t.name(), type_name_prefix));
-  auto path = t.name().substr(type_name_prefix.size());
+  auto path = std::string_view{t.name()};
+  auto i = path.find('.');
```

Comment:
Should this be `if (path.starts_with(type_name_prefix))`? Or do we actually want to cut arbitrary prefixes here?

Reply:
Example: input: `zeek.conn`
Desired output: `conn`

This is because the standard Zeek TSV format doesn't add `zeek` as we do to the type name. What we want is something like `split<1>(type_name, ".")[1]`, i.e., perform a single split at the first dot.

But now that I think about that, maybe we want to keep the full type name for all non-Zeek types? Or would that not be consistent? 

The Zeek output for Zeek data should definitely look as if Zeek produced it.

Reply:
`type_name_prefix` is defined as `"zeek."` currently, so for the example input both would produce the same result. The interesting questions are what happens when there's a dot that's not preceded by "zeek.", or when there are multiple dots. I don't know what the most reasonable behavior for these cases would be.

Reply:
>  The interesting questions are what happens when there's a dot that's not preceded by "zeek."

Right now, we always have the format name before the dot. This should work for all formats, though.

> [..] or when there are multiple dots.

It's currently implemented as "split-1", i.e., split on the first dot. If formats now contain a dot, we'll display some weird things.

Maybe the best logic would be: strip `zeek.` if the event starts like this, and use the literal event name otherwise.

Reply:
I've implemented as noted above. Does that make now sense to you @lava?

Reply:
Makes sense to me